src/injected/tab-stops-requirement-evaluator.ts

```diff
@@ -44,7 +44,7 @@ export class DefaultTabStopsRequirementEvaluator implements TabStopsRequirementE
         actualTabStops: Set<HTMLElement>,
     ): AutomatedTabStopRequirementResult[] {
         const requirementResults: AutomatedTabStopRequirementResult[] = [];
-        (tabbableTabStops as HTMLElement[]).forEach(expectedTabStop => {
+        ((tabbableTabStops as HTMLElement[]) ?? []).forEach(expectedTabStop => {
```

Comment:
I think it'd be better to use an if statement here since that's more readable.

Reply:
Also is this change necessary due to something in this PR or something else?

Reply:
With a different iteration of this fix I was noticing this method erroring when `tabbableTabStops` is null, so I figured I'd include the fix since it's so small. I don't think it's actually an issue anymore with this implementation of the fix